CMSIS/DSP/Include/arm_math.h

```diff
@@ -381,8 +381,8 @@ extern "C"
 #elif defined (__GNUC_PYTHON__)
 #include <stdint.h>
 #define  __ALIGNED(x) __attribute__((aligned(x)))
-#define __STATIC_FORCEINLINE static __attribute__((inline))
-#define __STATIC_INLINE static __attribute__((inline))
+#define __STATIC_FORCEINLINE static inline
```

Comment:
Force inline should do what it is supposed to do:

```suggestion
#define __STATIC_FORCEINLINE static inline __attribute__((always_inline))
```

Zephyr equivalent can be found here:
https://github.com/zephyrproject-rtos/zephyr/blob/fc03bd2b863d543bd9f2c87857b770f3de665cb6/include/toolchain/common.h#L116

Reply:
Thanks!